client/component/message.js

```diff
@@ -83,7 +83,7 @@ const QuietSystemMessageStyle = StyleSheet.create({
     text: {
         color: '#999',
         marginVertical: 8,
-        alignSelf: 'center'
+        alignSelf: 'left'
```

Comment:
This change will modify how other prototypes behave. In general, if you want a component to do something new for you, it's better to add an additional property, where the default value is however it behaved before. 

In this case, maybe you want to add a 'center' prop, whose default value is 'true', and then squirt in an 'alignSelf: center' into the style if that property is true.